README.md

```diff
@@ -16,8 +16,22 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+A CSS clearfix is used to fix issues related to floating child elements within a parent element
 ```
+Normally we would do this
+<div>
+    <div style="float: left;">Sidebar</div>
+    <div style="clear: both;"></div>
+</div>
+With Cleatfix we need only
+<div class="clearfix">
+    <div style="float: left;" class="clearfix">Sidebar</div>
+</div>
+.clearfix:after {
+  content: "";
+  display: table;
```

Comment:
I haven't seen it with `display: table` before; usually it's `display: block`. Cool!